src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -455,6 +455,21 @@ public GHTeam createTeam(String name, GHRepository... repositories) throws IOExc
         return createTeam(name, Arrays.asList(repositories));
     }
 
+    /**
+     * Starts a builder that creates a new team.
+     *
+     * <p>
+     * You use the returned builder to set various properties, then call {@link GHCreateTeamBuilder#create()} to finally
+     * create a team.
+     *
+     * @param name
+     *            the name
+     * @return the gh create repository builder
+     */
+    public GHCreateTeamBuilder createTeam(String name) {
```

Comment:
I think the other `createTeam` methods should be marked as `@Deprecated`.  Do you agree?

Reply:
Yes, wanted your input on it first before I did it, done.